operator/controllers/operation_controller.go

```diff
@@ -245,64 +243,71 @@ func (r *OperationReconciler) reconcileNormal(operation *operatorv1.Operation, l
 	operation.Status.FailedGroups = int32(len(taskGroups.failed))
 	operation.Status.InvalidGroups = int32(len(taskGroups.invalid))
 
+	return taskGroups, nil
+}
+
+func (r *OperationReconciler) reconcileNormal(operation *operatorv1.Operation, taskGroups *taskGroupReconcileList, log logr.Logger) error {
+	// If the Operation doesn't have finalizer, add it.
```

Comment:
is this code needed ? If not let's just deleted it, we can always add it back later

Reply:
[1] I'm still defining if we need finalizers or not.
I prefer to keep the comment around so I'm reminded about it